org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/RuntimeProjectDescriptor.xtend

```diff
@@ -344,10 +347,10 @@ class RuntimeProjectDescriptor extends TestedProjectDescriptor {
 							</configuration>
 							�IF config.needsTychoBuild�
 								<dependencies>
-									<dependency>
-										<groupId>org.eclipse.emf</groupId>
-										<artifactId>org.eclipse.emf.mwe2.launch</artifactId>
-										<version>�config.xtextVersion.mweVersion�</version>
+									<dependency>
+										<groupId>org.eclipse.emf</groupId>
+										<artifactId>org.eclipse.emf.mwe2.launch</artifactId>
+										<version>�config.xtextVersion.mweVersion�</version>
```

Comment:
The indentation format seems to have changed here (also below). Please check whether that has any consequences to the generated code.